<ui version="4.0" >
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Game Genie Encoder/Decoder</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>8</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Game Genie Encoder/Decoder 0.91a</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Copyright (C) 2004-2006,2008 emuWorks</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>http://games.technoplaza.net/</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>1</number>
       </property>
       <item row="1" column="1" >
        <widget class="QRadioButton" name="gbggRadio" >
         <property name="text" >
          <string>Game &amp;Boy / Game Gear</string>
         </property>
         <property name="shortcut" >
          <string>Alt+B</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QRadioButton" name="genesisRadio" >
         <property name="text" >
          <string>&amp;Genesis</string>
         </property>
         <property name="shortcut" >
          <string>Alt+G</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QRadioButton" name="snesRadio" >
         <property name="text" >
          <string>&amp;Super Nintendo</string>
         </property>
         <property name="shortcut" >
          <string>Alt+S</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QRadioButton" name="nesRadio" >
         <property name="text" >
          <string>&amp;Nintendo</string>
         </property>
         <property name="shortcut" >
          <string>Alt+N</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>5</number>
       </property>
       <item row="1" column="2" >
        <widget class="QLabel" name="textLabel4_2_2_2" >
         <property name="text" >
          <string>&amp;Compare</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>compareEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLineEdit" name="compareEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="textLabel2_2_2_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>&amp;Value</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>valueEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="textLabel3_2_2_2" >
         <property name="text" >
          <string>&amp;Address</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>addressEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="addressEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLineEdit" name="valueEdit" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>10</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_3_2" >
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="gamegenieEdit" />
         </item>
         <item>
          <widget class="QLabel" name="textLabel5_2_2_2_2_2" >
           <property name="text" >
            <string>Game G&amp;enie</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy" >
            <cstring>gamegenieEdit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>213</y>
     <width>359</width>
     <height>84</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_2" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>225</y>
     <width>97</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>123</y>
     <width>359</width>
     <height>84</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_5" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>135</y>
     <width>277</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="layoutWidget_3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>121</y>
     <width>385</width>
     <height>55</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_6" >
   <property name="geometry" >
    <rect>
     <x>66</x>
     <y>122</y>
     <width>272</width>
     <height>53</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_7" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>157</y>
     <width>447</width>
     <height>89</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_8" >
   <property name="geometry" >
    <rect>
     <x>94</x>
     <y>158</y>
     <width>278</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_9" >
   <property name="geometry" >
    <rect>
     <x>283</x>
     <y>159</y>
     <width>88</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_10" >
   <property name="geometry" >
    <rect>
     <x>189</x>
     <y>159</y>
     <width>88</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_11" >
   <property name="geometry" >
    <rect>
     <x>95</x>
     <y>159</y>
     <width>88</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget_4" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>71</y>
     <width>359</width>
     <height>46</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>nesRadio</tabstop>
  <tabstop>snesRadio</tabstop>
  <tabstop>genesisRadio</tabstop>
  <tabstop>gbggRadio</tabstop>
  <tabstop>valueEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>compareEdit</tabstop>
  <tabstop>gamegenieEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
